### PR TITLE
refactor: fix checkstyle in tests

### PR DESCRIPTION
**Pull request by zielint0**
- fixed checkstyle violations
- minor formatting fixes

I am ready to merge.

### CODE REVIEW

File: src/test/java/spoon/test/comment/CommentTest.java [hunk 1]
```diff
@@ -161,26 +163,26 @@ private void testJavaDocComment(CtClass type, String EOL) {
...
-		while(st.hasMoreTokens()) {
+		while (st.hasMoreTokens()) {
 			String line = st.nextToken();
-			if(first) {
+			if (first) {
 				//first
 				first = false;
-				assertTrue(line.length()==3);
-				assertEquals("/**", line); 
+				assertTrue(line.length() == 3);
```
Comment (surli) [inline src/test/java/spoon/test/comment/CommentTest.java]: You could have change it for an assertEquals here

File: src/test/java/spoon/test/comment/CommentTest.java [hunk 2]
```diff
@@ -842,20 +868,20 @@ public void testCommentsInComment1And2() {
...
+		try (InputStream is = new FileInputStream(new File("./src/test/java/spoon/test/comment/testclasses/WindowsEOL.java"))) {
 			int b;
 			boolean lastWasCR = false;
-			while((b = is.read())!=-1) {
-				if(lastWasCR) {
+			while ((b = is.read()) != -1) {
+				if (lastWasCR) {
 					//next must be LF
-					assertTrue(b=='\n');
+					assertTrue(b == '\n');
```
Comment (surli) [inline src/test/java/spoon/test/comment/CommentTest.java]: assertEquals

File: src/test/java/spoon/test/comment/CommentTest.java [hunk 3]
```diff
@@ -873,22 +899,22 @@ public void testCommentsInResourcesWithWindowsEOL() throws IOException {
...
-		while(st.hasMoreTokens()) {
+		while (st.hasMoreTokens()) {
 			String line = st.nextToken();
-			if(first) {
+			if (first) {
 				//first
 				first = false;
-				assertTrue(line.length()==3);
-				assertEquals("/**", line); 
+				assertTrue(line.length() == 3);
```
Comment (surli) [inline src/test/java/spoon/test/comment/CommentTest.java]: assertEquals

**Comment by zielint0** [comment]
@surli 
All implemented. Thank you!

**Pull request merged**